app/javascript/components/feedback-sessions-new.vue

```diff
@@ -0,0 +1,140 @@
+<script setup lang="ts">
+import { ref, onMounted } from 'vue';
+import VueTagsInput from '@sipec/vue3-tags-input';
+import api from 'api/feedback-sessions';
+import usersApi, { type UserForm } from 'api/users';
+import tagClass from '../utils/tag-class';
+
+interface VueTagsInputType {
+  text: string;
+  tiClasses: string[];
+  classes?: string;
+}
+
+const currentTagNames = ref<VueTagsInputType[]>([]);
+const newTagName = ref<string>('');
+const feedbackSessionDate = ref<string>('');
+const providerUserId = ref<number>(0);
+const allUsers = ref<{ id: number; name: string }[]>([]);
+
+function capitalizeString(input: string) {
+  const trimmedInput = input.trim();
+
+  return trimmedInput.charAt(0).toUpperCase() + trimmedInput.slice(1).toLowerCase();
+}
+
+function handleTagsChanged(newTags: VueTagsInputType[]) {
+  if (currentTagNames.value.length > newTags.length) {
+    currentTagNames.value = newTags;
+  } else {
+    const newTag = newTags.pop();
+    if (!newTag) return;
+    newTag.text = capitalizeString(newTag.text);
+    newTag.classes = tagClass(newTag.text);
+    currentTagNames.value.push(newTag);
+  }
+}
+
+async function createFeedbackSession() {
+  const response = await api.create(
+    providerUserId.value,
+    feedbackSessionDate.value,
+    currentTagNames.value.map(obj => obj.text),
+  );
+  const createdStatus = 201;
+  if (response.status === createdStatus) {
+    window.location.href = '/';
+  }
+}
+
+async function fetchUsers() {
+  const res = await usersApi.index();
```

Comment:
🚫 **[tsc]** <2339> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property 'index' does not exist on type '{ login(email: string, password: string, remember: boolean): Promise<AxiosResponse<any, any>>; register(user: UserForm): Promise<AxiosResponse<any, any>>; }'.